.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
   - "node"
-  - "8"
```

Comment:
Some of the test implementations use `Array#flatMap` which this old NodeJS version does not support.